cfme/tests/infrastructure/test_provisioning_dialog.py

```diff
@@ -83,7 +89,7 @@ def _provisioner(template, provisioning_data, delayed=None):
         fill(provisioning_form, provisioning_data, action=provisioning_form.submit_button)
         flash.assert_no_errors()
 
-        request.addfinalizer(lambda: cleanup_vm(vm_name, provider))
+        request.addfinalizer(lambda: remove_vm(vm_name, provider))
```

Comment:
Well, it somehow works as a workaround for some of the tests (like test_provision from provisioning_rest), so can we just keep this open for now? As I already said in some comment, I have no intent to get this PR merged, it's just here as a temporary workaround that we check out in jenkins